src/js/components/Heading/doc.js

```diff
@@ -17,7 +17,7 @@ export const doc = (Heading) => {
       `A color identifier to use for the text color. For example:
       'brand'.`
     ),
-    level: PropTypes.oneOf([1, 2, 3, 4, '1', '2', '3', '4']).description(
+    level: PropTypes.number.description(
```

Comment:
So, this will be a breaking change if a level is specified as a string. For example: `<Heading level="3" />` will show a warning. I am OK with it being just `number`, but we should add a note in the compatibility section of the PR.

Reply:
oh no that was unintentional. let me change to number or any string.